include/ibamr/INSVCStaggeredConservativeConvectiveOperator.h

```diff
@@ -275,31 +299,35 @@ class INSVCStaggeredConservativeConvectiveOperator : public ConvectiveOperator
         const double& dt,
         const double* const dx);
 
+    /*!
+     * \brief Enforce divergence free condition at the coarse-fine interface to ensure conservation of mass.
+     */
+    void enforceDivergenceFreeConditionAtCoarseFineInterface(const int U_idx);
```

Comment:
Is this something that also should be done in the constant-coefficient Navier-Stokes solver?

Reply:
Maybe.

Reply:
It prevents density fluctuations and keeps it bounded. Same is true at physical BCs. Hence, we do it for all BCs.